config/routes/Centreon/configuration/contact_template.yaml

```diff
@@ -0,0 +1,5 @@
+configuration_contact_template_find_all:
+  methods: GET
+  path: /configuration/contact/template
```

Comment:
```suggestion
  path: /configuration/contacts/templates
```

Reply:
I think we should discuss about this, as in my opinion a GET on contact/template without any id or parameters is plural as we need to get All, if i have /contact/template/{id} I need one. The endpoint should reflect the resource type but i'm totally open to discuss about it !